packages/core/src/auth/auth.ts

```diff
@@ -502,8 +506,10 @@ export class Auth implements AuthService, ConnectionManager {
     private async validateConnection<T extends Profile>(id: Connection['id'], profile: StoredProfile<T>) {
         const runCheck = async () => {
             if (profile.type === 'sso') {
+                getLogger().info(`Validating SSO connection ${id}`)
```

Comment:
why are these info instead of debug? this will cause a lot of messages for all users.

Reply:
Because level info is by default. Many users do not adjust log level before providing log to us, making it hard for us to get debug level logs. 

Making them info level is a temporary solution(actually proposed by Anil). We can downgrade them to debug once this auth issue is resolved.